repo/connect.go

```diff
@@ -17,11 +17,17 @@ import (
 
 // ConnectOptions specifies options when persisting configuration to connect to a repository.
 type ConnectOptions struct {
+	PersistCredentials bool `json:"persistCredentials"`
+
 	content.CachingOptions
 }
 
 // Connect connects to the repository in the specified storage and persists the configuration and credentials in the file provided.
-func Connect(ctx context.Context, configFile string, st blob.Storage, password string, opt ConnectOptions) error {
+func Connect(ctx context.Context, configFile string, st blob.Storage, password string, opt *ConnectOptions) error {
```

Comment:
@jkowalski Just curious, what's the rationale for using a pointer for `opt`?

Reply:
linter was saying it was too big...

Reply:
got it